src/http_rest_api_handler.cpp

```diff
@@ -242,149 +242,6 @@ void HttpRestApiHandler::parseParams(Value& scope, Document& doc) {
-    if (input.datatype() == "FP32") {
-        if (input.contents().fp32_contents_size() > 0) {
-            SPDLOG_DEBUG("FP32" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "INT64") {
-        if (input.contents().int64_contents_size() > 0) {
-            SPDLOG_DEBUG("INT64" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "INT32") {
-        if (input.contents().int_contents_size() > 0) {
-            SPDLOG_DEBUG("INT32" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "INT16") {
-        if (input.contents().int_contents_size() > 0) {
-            SPDLOG_DEBUG("INT16" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "INT8") {
-        if (input.contents().int_contents_size() > 0) {
-            SPDLOG_DEBUG("INT8" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "UINT64") {
-        if (input.contents().uint64_contents_size() > 0) {
-            SPDLOG_DEBUG("UINT64" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "UINT32") {
-        if (input.contents().uint_contents_size() > 0) {
-            SPDLOG_DEBUG("UINT32" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "UINT16") {
-        if (input.contents().uint_contents_size() > 0) {
-            SPDLOG_DEBUG("UINT16" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "UINT8") {
-        if (input.contents().uint_contents_size() > 0) {
-            SPDLOG_DEBUG("UINT8" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "FP64") {
-        if (input.contents().fp64_contents_size() > 0) {
-            SPDLOG_DEBUG("FP64" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else if (input.datatype() == "BYTES") {
-        if (input.contents().bytes_contents_size() > 0) {
-            SPDLOG_DEBUG("BYTES" CONTENT_FIELD_NOT_EMPTY_ERROR_MESSAGE);
-            return StatusCode::REST_CONTENTS_FIELD_NOT_EMPTY;
-        }
-    } else {
-        return StatusCode::REST_UNSUPPORTED_PRECISION;
-    }
-
-    return StatusCode::OK;
-}
-
 static bool isInputEmpty(const ::KFSRequest::InferInputTensor& input) {
```

Comment:
What happens if we have input of datatype FP32 but there is some content in int64_contents?